readthedocs/templates/projects/notifications/deprecated_config_file_used_email.html

```diff
@@ -13,7 +13,8 @@
     <li><strong>Monday, September 25, 2023</strong>: Fully remove support for building documentation without configuration file v2.</li>
 </ul>
 
-We have identified that the following projects which you maintain are impacted by this deprecation:
+We have identified that the following projects which you maintain, are impacted by this deprecation.
+Note that we changed our policy for sending this deprecation email to only projects with a successful build in the last year.
```

Comment:
Any reason we need to tell the user this at all? This change mostly affects users that won't get the notification emails either way.

Reply:
We told all the users that they will receive weekly notification for that list of projects if they don't upgrade. Next wednesday, we will be sending a different list of projects omitting projects that haven't been migrated yet --they are not listed in the new email because they don't have a build in the last year. That was my reason, to avoid confusion basically.